deployer/conf/elasticsearch-logging.yml

```diff
@@ -9,6 +9,8 @@ logger:
   io.fabric8.elasticsearch: ${PLUGIN_LOGLEVEL}
   io.fabric8.kubernetes: ${PLUGIN_LOGLEVEL}
 
+  io.fabric8.elasticsearch.plugin.acl.DynamicACLFilter: DEBUG
```

Comment:
This is just for testing... getting an warning from our client and not sure what's the source. That is going to be reverted :)